### PR TITLE
Nerfs overall first aid kits, and compensates the nerfs with some touches

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
follow-up PR with https://github.com/BeeStation/BeeStation-Hornet/pull/7936
* adds tricordrazine pills to first aid kits
* adjust medical basic kits
* compensates some toxin cases for the medbay since it'd be difficult to manage with default ones. (antitoxin syringe is currently awful for this state, so buff it a bit)
* retractor is now default to the doctor's box. That's because I removed one brute pack and one ointment!

## Why It's Good For The Game
first aid kits are too strong. let's make it less-powergamy with awfully nerfed medicine pills.

basic medical kit with 'basic medicine' will be nightmare for right now, so lets buff it a little.
2u of advanced chem is enough to heal 30 damage

there are a few compensation since the nerf is a bit impactful.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

image

-------------

image

image

first aid one only has 4 pills. (increased 3 to 4 later)

------------

image

----------

image

</details>

## Changelog
:cl:
add: a pre-made pill 'Pain Relief' that has 36u of tricordrazine, and its pill bottle. (Note: Tricordrazine effect was changed.)
add: a pre-made pill 'brute-treatment' that has bicaridine 10u + salicyclic acid 2u.
add: a pre-made pill 'burn-treatment' that has which contains kelotane 10u + oxandrolon 2u.
balance: medical roundstart brute kit's pill bottle is now filled with 7 of 'brute-treatment' pills
balance: medical roundstart burn kit's pill bottle is now filled with 7 of 'burn-treatment' pills
balance: medical roundstart toxin kit's antitoxin syringe is now filled with antitoxin(10.5u) + carthatoline(4.5u).
balance: all types of first aid kits have one less bruise pack and one less ointment than before, and these will get a pill bottle of 'Pain Relief' pills, and one additional epipen instead. (ancient first aid kit takes bicaridine/kelotane pill bottle instead)
tweak: doctors will have a retractor in their roundstart medikit (since they have less brute pack and ointment)
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -446,6 +451,18 @@
...
+	name = "bottle of Washes Pain Away pills"
+	desc = "A pill bottle that contains pills that tend all small damages. Its label says: 'Supplement Facts: Tricordrazine 36u. WARN: Do NOT eat more than one pill.'"
+
+/obj/item/storage/pill_bottle/tricordrazine_first_aid/PopulateContents()
+	for(var/i in 1 to 3)
+		new /obj/item/reagent_containers/pill/tricordrazine_first_aid(src)
+
+/obj/item/storage/pill_bottle/tricordrazine_first_aid/medical_doctor/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/reagent_containers/pill/tricordrazine_first_aid(src)
```
Comment (EvilDragonfiend) [inline code/game/objects/items/storage/firstaid.dm]: I know this hard coding is awful, but please understand that this PR's main purpose is not refactoring these codes.

File: code/modules/reagents/reagent_containers/pill.dm [hunk 2]
```diff
@@ -162,6 +162,13 @@
 	list_reagents = list(/datum/reagent/medicine/kelotane = 15)
 	rename_with_volume = TRUE
 
+/obj/item/reagent_containers/pill/tricordrazine_first_aid
+	name = "Washes Pain Away pill"
+	desc = "A simple pill that tends all small damages. You might need to read an drug instruction in the pill bottle..."
```
Comment (CydiaLamiales) [inline code/modules/reagents/reagent_containers/pill.dm]:
```suggestion
	name = "Pain Relief pill"
	desc = "A pill that cures basic ailments. You feel like you should read the instructions on the pill bottle again..."
```

File: code/modules/reagents/reagent_containers/pill.dm [hunk 3]
```diff
@@ -162,6 +162,13 @@
...
+	icon_state = "pill_happy"
+	list_reagents = list(/datum/reagent/medicine/tricordrazine = 36)
```
Comment (CydiaLamiales) [inline code/modules/reagents/reagent_containers/pill.dm:171]: Why do they have 36u each when 30u is the OD limit for tricordrazine?

File: code/game/objects/items/storage/firstaid.dm [hunk 4]
```diff
@@ -446,6 +451,18 @@
 	for(var/i in 1 to 7)
 		new /obj/item/reagent_containers/pill/kelotane(src)
 
+/obj/item/storage/pill_bottle/tricordrazine_first_aid
+	name = "bottle of Washes Pain Away pills"
+	desc = "A pill bottle that contains pills that tend all small damages. Its label says: 'Supplement Facts: Tricordrazine 36u. WARN: Do NOT eat more than one pill.'"
```
Comment (CydiaLamiales) [inline code/game/objects/items/storage/firstaid.dm]:
```suggestion
	name = "bottle of Pain Relief pills"
	desc = "A pill bottle containing basic healing medicine. The label on it says: 'Total contents: Tricordrazine 36u x7. WARNING: Do NOT take more than one pill.'"
```

**Comment by CydiaLamiales** [comment]
Dangit, the reviews came out a bit messy. Soz, first time doing this.

File: code/modules/reagents/reagent_containers/pill.dm (hunk 3 above)
Comment (EvilDragonfiend) [reply code/modules/reagents/reagent_containers/pill.dm:171]: because its OD is now 50u and tricordrazine is pretty weak now.

Comment (CydiaLamiales) [reply code/modules/reagents/reagent_containers/pill.dm:171]: Oh right, that other PR just got merged.

**Comment by PowerfulBacon** [comment]
I don't really like this, it feels like a bit of band-aid over the medical system being easy. I dislike the basic medkits only having a single of each healing thing, since thats only enough to heal 1 person's injuries before needing to grab another.
Medkits are pretty strong if you keep one for yourself, but if you are a doctor and need to heal a lot of people then they start to deplete pretty quick. The PR that makes medkits large is a decent idea for a solution.

**Comment by EvilDragonfiend** [comment]
gonna keep the conflict. Feel free to remain the opinion about the balance

**Comment by PowerfulBacon** [comment]
This PR is supersceded by https://github.com/BeeStation/BeeStation-Hornet/pull/8572 closing in favour of

**Pull request closed without merging**